src/stringToMpdXml.js

```diff
@@ -0,0 +1,19 @@
+import window from 'global/window';
+
+export const stringToMpdXml = (manifestString) => {
+  if (manifestString === '') {
+    throw new Error('DASH_EMPTY_MANIFEST');
```

Comment:
We should make sure to unit test this and the below to ensure the error gets thrown.  It also might not be bad to extract these error strings somewhere else (you'll be able to consume them in the test and it will limit the number of magic strings).